bot/core/events/handlers.py

```
@@ -0,0 +1,227 @@
+"""
+Global event handlers for the bot.
+
+This module provides predefined event handlers for common scenarios and system events.
+"""
+
+import logging
+import traceback
```

Ruff (0.8.2)

`traceback` imported but unused

Remove unused import: `traceback`

(F401)